spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/streaming/StreamingContextFunctions.scala

```diff
@@ -17,14 +17,13 @@ import scala.reflect.ClassTag
 class StreamingContextFunctions (ssc: StreamingContext) extends SparkContextFunctions(ssc.sparkContext) {
   import scala.reflect.ClassTag
 
-  override def cassandraTable[T](keyspace: String, table: String)(
+  override def cassandraTable[T](keyspace: String, table: String, readConf: ReadConf = ReadConf.fromSparkConf(ssc.sparkContext.getConf))(
```

Comment:
please move `readConf` parameter to the implicits section (in all `cassandraTable` declaractions)